x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -489,6 +489,7 @@ class PackagePolicyService {
             title: packagePolicy.package?.title || '',
             version: packagePolicy.package?.version || '',
           },
+          policy_id: packagePolicy.policy_id,
```

Comment:
This is something that we need clarification on. Thanks!

Reply:
What exactly needs clarification here? Is the intent to expose this SO field via the Fleet REST API?

Reply:
Hey Kyle, 
Firstly, thanks for finding time to check this out :) 
Secondly, sorry I should have explained it more explicitly. 
The thing is - are you ok with us adjusting this response? Isn't this against your initial plan for this interface?

We need the policy_id to be able to remove agent from Packs in Osquery. 
Thanks!

Reply:
Understood - thanks for clarifying. I think this is fine to expose here - not sure why it wasn't exposed in the first place. Thanks for reaching out to clarify.